docs/quickstart.md

```diff
@@ -0,0 +1,228 @@
+Now, let's start the jorney with NPLinker by following the steps below:
+
+[TOC]
+
+
+## 1. Decide which mode to use
+NPLinker allows you to run in two modes:
+
+=== "`local` mode"
```

Comment:
I really like these two tabs thing (local mode vs podp mode). But I think that the instructions flow may be further improved. It can be confusing to read here the two tabs (local/podp), but later `3. Prepare input data` is meant for local mode only. I thought about two alternatives:
1. `1. Decide which mode to use` title may be removed, and that part becomes an introduction not numbered, with `2. Create a working directory` becoming `1. Create a working directory`. There will be then `2. Prepare input data (local mode only)`. The numbering would make more sense in my opinion. 
2. With the local/podp tabs, we update all the numbered list accordingly. Which means, for the local mode tab, we would have `2. Create a working directory`, `3. Prepare input data`, `4. Prepare config file` and so on. For the podp mode tab, we would have `2. Create a working directory`, `3. Prepare config file`, and so on.

Reply:
I considered the idea of totally separating the two tabs before but did not accept it for the following reasons:
- it will lead to lots of replications of source text in the file since most steps are shared by `local` and `podp`, which will then cause much maintenance burden
- the structure of TOC will be confusing, e.g. there will be two `2. Create a working directory` in the TOC.

I updated it a bit to collapse the local mode only step, hope the new layout making it more logical.

Reply:
Nice! I think is much clearer now :)